pixart.js

```diff
@@ -1 +1,52 @@
+var clickCount = 0;
```

Comment:
I'm guessing this for debugging purposes, but if not: why did you define these variables outside of `document.ready( )`?